Documentation/kubernetes-on-vagrant-single.md

```diff
@@ -57,19 +57,19 @@ You can choose from one of the two following options.
 
 1. **Use a custom KUBECONFIG path**
 
-   ```sh
-   $ export KUBECONFIG="${KUBECONFIG}:$(pwd)/kubeconfig"
-   $ kubectl config use-context vagrant-single
-   ```
+```sh
+$ export KUBECONFIG="${KUBECONFIG}:$(pwd)/kubeconfig"
+$ kubectl config use-context vagrant-single
+```
 
 1. **Update the local-user kubeconfig**
```

Comment:
Can you add a `2\.` here? The indentation was what triggered the automatic numbering. This will force it to be a 2.